### ISSUE TITLE
The column headers of the grid present in the "Result" section are not accessible via keyboard keys: A11y_mssql for VSCode_MS Sql Extension_Execute Pre-built Query_Keyboard

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/vscode-mssql#18657]
**Github Tags:** #A11yTCS; #A11ySev2; #A11yMAS; #WCAG2.1.1; #Keyboard; #FTP; #Win32; #Win11; #DesktopApp; #MSSQL for VSCode; #BM_MSSQL for VSCode_Win32_Feb2025; #AILimited; #Rev:shta;

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification** 

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Environment Details:

Application Name: MSSQL for VSCode
Windows 11 24H2 OS Build 26100.3037

## Repro Steps:

1. Launch VS Code Application.
2. Open SQL Server Extension and connect to authenticated SQL Server.
3. In the "Connections" section, expand server>Database>System Databases> master>Tables>System tables>open context menu.
4. Use arrow key, and press enter on SELECT TOP 1000.
5. The selected query will provide the result in a grid.
6. Tab to the grid present in "Result" section.
7. Observe the column headers are not accessible via keyboard keys.
 
## Actual Results: 
The column headers of the grid present in the "Result" section are not accessible via keyboard keys.

## Expected Results:
The column headers in the "Result" section should be accessible via keyboard keys, allowing users to navigate and interact with the interactive elements present in the column headers.

## User Impact:
This issue affects users who rely on keyboard navigation, making it difficult or impossible for them to interact with the column headers in the "Result" section.

## WCAG Reference:  

https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html

## Attachments:

![Image](https://github.com/user-attachments/assets/12d77560-d6a6-411f-8e79-f9ef7038cea7)

**Comment by laurenastrid1** [comment]
@ManishaC1   The fix for this has been merged! Please verify the fix using the August release. Thanks!
